Update README.md

kellysmith1008 opened the pull request:
Updated the design practice area guidebook information

<!-- readthedocs-preview civicactions-handbook start -->
----
📚 Documentation preview 📚: https://civicactions-handbook--1366.org.readthedocs.build/en/1366/

<!-- readthedocs-preview civicactions-handbook end -->

kellysmith1008 committed:
Update README.md

Updated the design practice area guidebook information

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra commented on practice-areas/design-and-research/README.md:
```diff
@@ -1,41 +1,102 @@
+# Design practice area
 
-## We start with discovery, not solutions
+The CivicActions design team works with our clients on projects and partners with our other [internal practice areas](https://guidebook.civicactions.com/en/latest/practice-areas/). Our design area includes content designers, product designers, design educators, and service designers. We use the **#just-design** slack channel for our communications and the **#design** slack channel to communicate  ...
```
@kellysmith1008 could we make the link for 'internal practice areas' relative? Here is an example of the same link on the Drupal page https://github.com/CivicActions/guidebook/blob/master/practice-areas/engineering/drupal/README.md?plain=1#L7. I believe for this page the link would be:

`[internal practice areas](../../practice-areas/README.md)`

kellysmith1008 replied:
@dmundra We sure can! Well, you can, I need to learn how! Are you saying to replace the guidebook links with ../../../{page-name}/README.md links instead? Sorry if that's a silly question, but want to make sure I understand before I change it (if I can figure out how to change it!)

dmundra replied:
> Are you saying to replace the guidebook links with ../../../{page-name}/README.md links instead?

Yup exactly. I fixed my first comment to show the code snippet correctly. Sorry about that.

> Sorry if that's a silly question, but want to make sure I understand before I change it (if I can figure out how to change it!)

Not a silly question.

kellysmith1008 replied:
@dmundra Ok, I think I made both changes, but please let me know if I did it incorrectly!

dmundra replied:
The changes look great. I will comment on the one gotcha for the professional development link.

dmundra commented on practice-areas/design-and-research/README.md:
```diff
@@ -1,41 +1,102 @@
+
+## Professional development
+
+We've included some possible professional development resources below. You have $1200 per year (after your 90-day intro period) to spend on [professional development](https://guidebook.civicactions.com/en/latest/employee-benefits/professional-development/). Professional development conferences, courses, and webinars that align with your current or growth-level path can contribute to your yearly u ...
```
Here is another link that could use a relative path

dmundra reviewed (changes requested):
Thanks @kellysmith1008. Small changes to a couple of links. Let me know if I can help with those changes.

kellysmith1008 committed:
Update README.md

Updated relative links per suggestion from Daniel

dmundra commented on practice-areas/design-and-research/README.md:
```diff
@@ -1,41 +1,102 @@
+
+## Professional development
+
+We've included some possible professional development resources below. You have $1200 per year (after your 90-day intro period) to spend on [professional development](../../professional-development/README.md). Professional development conferences, courses, and webinars that align with your current or growth-level path can contribute to your yearly utilization targets, but you don't have to use yo ...
```
@kellysmith1008 ah this one is getting caught in the problem because the file is `professional-development.md` instead of a README path.

dmundra replied:
Sorry for the trouble here @kellysmith1008. I believe the full link to that file is:

`[professional development](../../employee-benefits/professional-development.md)`

kellysmith1008 replied:
Ah crapballs. I'll get there eventually! Thank you for your patience and help!

dmundra replied:
No worries. I believe you got it.

kellysmith1008 committed:
Update README.md

Updated the professional development relative path to remove README and end it with professional-development.md

kellysmith1008 committed:
Update README.md

Updated the professional development link to ../../employee-benefits/professional-development.md - I still had the incorrect link last time

dmundra reviewed (approved):
@kellysmith1008 sounds good. Approved, feel free to merge it.
